code/modules/clothing/spacesuits/rig/suits/station.dm

```diff
@@ -519,3 +519,46 @@ Technomancer RIG
 		/obj/item/rig_module/ai_container,
 		/obj/item/rig_module/storage/large
 		)
+
+/***************************************
+	MU66 Suit
+****************************************/
+/obj/item/rig/sea
+	name = "MU-66 Atmos suit"
+	suit_type = "MU66 Soft-suit"
+	desc = "A light RIG for performing minute repairs and maintenance in atypical atmospheres."
+	icon_state = "sea_rig"
+	armor_list = list(
+		melee = 40,
```

Comment:
these are numbers for the old armor system. This suit will make you near indestructible please look at the other suits for examples.

Reply:
infact same goes for **all** the armors you have added